Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.



Generated by build [OMERO-DEV-latest-docs-autogen#1149](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1149/).

----
--no-rebase

hflynn commented:
This will be green once https://github.com/openmicroscopy/openmicroscopy/pull/5630 is merged and the job is re-run

joshmoore commented:
Hmmm.... merged and ran https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1150 but the update here still has "extensions".

hflynn commented:
Ah! I think the code build needs to run again before the changes can be ported across. Should be green tomorrow morning.

hflynn commented:
Yeah, safest not to merge until all the code PRs are merged and the code build re-run just incase there's something else to come across in the configs or whatever.

hflynn commented on omero/users/history.rst:
```diff
@@ -5,6 +5,55 @@
+-  added instructions to :doc:`/developers/cli/extending` on how to
+   create CLI plugins that are ``pip`` installable
+-  substantial effort to make third-party repositories easily testable;
+   see `omero-test-infra <https://github.com/openmicroscopy/omero-test-infra>_`
```
this is malformed and doesn't work in the built docs - see https://github.com/openmicroscopy/openmicroscopy/pull/5632 for the fix

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#1155 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1155/)
